app/assets/javascripts/course/assessment/test_case.js

```diff
@@ -0,0 +1,18 @@
+//= require jquery-shorten/src/jquery.shorten
+
+(function($) {
+  'use strict';
+  var DOCUMENT_SELECTOR = '.expected';
```

Comment:
This should be the the controller action selectors in that page. You can use our other js code as a reference. Otherwise your js is adopted to every page of the entire app.